doc/LOGIN_SYSTEMS.md

```diff
@@ -56,6 +59,14 @@ Or, write them in config/application.yml file
 Add the app_id and app_secret in the Jenkins and containers/docker*.yml files in the production.
 They are accessed by ENV["OAUTH_GITHUB_APP_KEY"], ENV["OAUTH_GITHUB_APP_SECRET"] etc inside the (config/initializers/omniauth.rb)[https://github.com/publiclab/plots2/blob/master/config/initializers/omniauth.rb]
 
+## How to setup login on various locations?
+
+For better UI on the website, we recently created login and signup modals. The corresponding code is present here https://github.com/publiclab/plots2/blob/master/app/views/layouts/_header.html.erb#L176-L266.
```

Comment:
We don't write amigious words like here in the documentations. please remove it.